CHANGELOG.md

```diff
@@ -8,12 +8,30 @@
 
 ## Unreleased
 
+## Fixes
+
+- Ensure IP address is only inferred by Relay if `sendDefaultPii` is `true` ([#5138](https://github.com/getsentry/sentry-react-native/pull/5137))
+
 ### Dependencies
 
 - Bump Bundler Plugins from v4.2.0 to v4.3.0 ([#5131](https://github.com/getsentry/sentry-react-native/pull/5131))
   - [changelog](https://github.com/getsentry/sentry-javascript-bundler-plugins/blob/main/CHANGELOG.md#430)
   - [diff](https://github.com/getsentry/sentry-javascript-bundler-plugins/compare/4.2.0...4.3.0)
 
+### Important Changes
```

Comment:
nit: should we move this above the `Fixes` section?

Reply:
sounds good to me.